composer.json

```diff
@@ -46,6 +46,7 @@
     "drupal/length_indicator": "^1.0",
     "drupal/libraries": "3.0-alpha1",
     "drupal/linkit": "~5.0",
+    "drupal/login_onlyemail": "^1.5",
```

Comment:
I have used docker images for EzContent installation. What is the default email and password for log in?

The installation steps is exact as (https://hub.docker.com/r/srijanlabs/ezcontent) below:

Create a network
docker network create ez-nw

Create containers
docker run --rm --name mysql -e MYSQL_ROOT_PASSWORD=drupal -e MYSQL_USER=drupal -e MYSQL_PASSWORD=drupal -e MYSQL_DATABASE=drupal --network ez-nw -p 3306:3306 -d mysql:5.7
docker run --rm --name ezcontent_drupal --network ez-nw -e DB_HOST=mysql -e DB_USER=drupal -e DB_PASSWORD=drupal -e DB_PORT=3306 -e DB_NAME=drupal -p 8062:8080 -d srijanlabs/ezcontent:8.x
Run following command to install the Ezcontent Stack
docker exec -it ezcontent_drupal sh -c "php -d memory_limit=-1 ./vendor/bin/drush si ezcontent --db-url='mysql://drupal:drupal@mysql/drupal' --site-name='EZ Content' --account-name='srijan' --account-pass='admin12345' -y && php -d memory_limit=-1 ./vendor/bin/drush en ezcontent_demo -y && php -d memory_limit=-1 ./vendor/bin/drush en ezcontent_api -y && php -d memory_limit=-1 ./vendor/bin/drush en ezcontent_api -y"